alf/algorithms/mbrl_algorithm.py

```diff
@@ -136,6 +138,9 @@ def __init__(self,
         self._planner_module = planner_module
         self._planner_module.set_action_sequence_cost_func(
             self._predict_multi_step_cost)
+        if dynamics_module is not None:
+            self._num_dynamics_replicas = dynamics_module._num_replicas
```

Comment:
In general, we should directly access the private member of a module, which are those prefixed with '_'

Reply:
Now switched to using a property function.